torch/distributed/algorithms/ddp_comm_hooks/default_hooks.py

```diff
@@ -33,6 +33,15 @@ def allreduce_hook(
     """
     return _allreduce_fut(process_group, bucket.get_tensors()[0])
 
+def decompress_fp16_callable(bucket: dist.GradBucket, world_size: int) -> callable:
```

Comment:
No need to refactor the code and create this util function, because this util function divide the output by `world_size`. Such division should not be used by `fp16_compress_wrapper`, because it's the responsibility of the hook inside the wrapper.